transform/mattermost-analytics/models/staging/mm_telemetry_prod/_mm_telemetry_prod__models.yml

```diff
@@ -229,3 +229,101 @@ models:
         description: The name of Rudderstack library used to send this event.
       - name: sent_at
       - name: original_timestamp
+
+  - name: stg_mm_telemetry_prod__license
+    description: |
+     Telemetry containing license related data. Telemetry source is server-side components via Rudderstack.
+    columns:
+      - name: event_id
+        description: The event's id.
+      - name: event_name
+        description: The name of the event.
+      - name: event_table
+        description: The name of the event table.
+      - name: server_id
+        description: The ID of the server the event originated from.
+      - name: received_at
+        description: Timestamp registered by Segment when the event was ingested (received).
+      - name: timestamp
+        description: Timestamp registered by Segment when the event occured refactored to clients clock skew.
+      - name: license_id
+        description: The id of the license used at the time the event was sent.
+      - name: customer_id
+        description: The customer ID, as defined in the license.
+      - name: installation_id
+        description: The id of the installation
+      - name: anonymous_id
+        description: Server's anonymous ID.
+      - name: license_name
+        description: The name of the current license.
+      - name: licensed_seats
+        description: The number of licensed seats for the current license.
+      - name: issued_at
+        description: The date and time the license was issued at.
+      - name: start_at
+        description: The date and time the license started at.
+      - name: expire_at
+        description: The date and time the license expires at.
+      - name: is_feature_advanced_logging_enabled
+        description: Whether feature "feature_advanced_logging" is enabled.
```

Comment:
nit: keep the same description template like for `stg_mattermost2__license` - e.g. 
`Whether feature "feature_cloud" is enabled`. -> `Whether feature "cloud" is enabled.`

Reply:
Agree. Converted to same template.